test/lib/infer/params.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var test = require('tap').test,
+  parse = require('../../../lib/parsers/javascript'),
+  inferParams = require('../../../lib/infer/params')();
+
+function toComment(fn, file) {
+  return parse({
+    file: file,
+    source: fn instanceof Function ? '(' + fn.toString() + ')' : fn
+  })[0];
+}
+
+function evaluate(fn, file) {
+  return inferParams(toComment(fn, file));
+}
+
+test('inferParams', function (t) {
+  t.deepEqual(evaluate(function () {
+    /** Test */
+    function f(x) {}
+  }).params, [{lineNumber: 3, name: 'x', title: 'param'}]);
+
+  t.deepEqual(evaluate(`
```

Comment:
I wasn't sure if you allowed template literals? If not I can change this to a string literal as needed.